docker-context/Dockerfile

```diff
@@ -30,25 +30,37 @@ RUN { \
 		echo 'memory_limit = 1024M'; \
 	} > /usr/local/etc/php/conf.d/tap.ini;
 RUN wp --allow-root package install https://github.com/nlemoine/wp-cli-fixtures/archive/refs/tags/0.3.0.zip
+COPY apache_port_template.conf /etc/apache2/ports.conf
+# Make sure the port override below matches the docker-compose port for wp49-php72
+RUN sed -i 's/<PORT>/8031/g' /etc/apache2/ports.conf
 COPY . /tmp/docker-context
 
 FROM wordpress:5.9-php7.4 AS wp59-php74
 COPY --from=wordpress:cli-php7.4 /usr/local/bin/wp /usr/local/bin/wp
 COPY --from=builder /root/.wp-cli /root/.wp-cli
 COPY --from=builder /tmp/base-theme.zip /tmp/base-theme.zip
 COPY --from=builder /tmp/wpamp-plugin.zip /tmp/wpamp-plugin.zip
+COPY apache_port_template.conf /etc/apache2/ports.conf
+# Make sure the port override below matches the docker-compose port for wp59-php74
```

Comment:
Probably worth to define these as `WP59_PHP74_PORT` in `.env` file and reuse it in `Dockerfile` and `docker-compose.yml`